docs/HOWTO_INTEGRATE_BANDWIDTH.md

```diff
@@ -1,6 +1,6 @@
 # Bandwidth Integration
 
-Bandwidth.com is a telephone service API company. To use Bandwidth, set `DEFAULT_SERVICE=bandwidth`. The `sticky-sender` and `num-picker` service managers are required for the Bandwidth extension to work. `sticky-sender` must come before `numpicker-basic` in the `SERVICE_MANAGERS` environment variable.
+Bandwidth.com is a telephone service API company. To use Bandwidth, set `DEFAULT_SERVICE=bandwidth`. The `sticky-sender` and `numpicker-basic` service managers are required for the Bandwidth extension to work. `sticky-sender` must come before `numpicker-basic` in the `SERVICE_MANAGERS` environment variable. NOTE: Because service managers are complicated, there is not an entry for SERVICE_MANAGERS in the Spoke Documentation Hub. Just add it as an enviromental variable.
```

Comment:
Here's the documentation for [Service Managers](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO-use-service-managers.md)

We could have the Service Managers doc (linked above) can have a new section for `ngpvan` and beneath it, link the HOWTO Integrate Bandwidth.  I'd be happy to make those changes and push them to this branch. :)

Reply:
I will look at what is required to update the Service Managers doc. Please feel free to close the request :)